gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -13,7 +13,7 @@ const { validationErrorHandler } = require('../../utils/helpers');
  * if change true, changes 'cancel' + 'back' button href back to application preview
  * else renders back to unissued facilities list
  * @param {req} params, @param {req} query, @param {Boolean} change
- * @returns {Object} body
+ * @returns {Promise<object>} body
```

Comment:
I know the above isn't part of this PR, but `req` and `Boolean` do not appear to be the correct types (same for the below docstrings)

Reply:
am simply going to remove these for now -- as they are incorrect but working out types is beyond the scope